src/OVAL/probes/SEAP/seap-descriptor.h

```diff
@@ -82,7 +80,7 @@ typedef struct {
 #define SEAP_MAX_OPENDESC 128
 #define SDTABLE_REALLOC_ADD 4
 
-int          SEAP_desc_add (SEAP_desctable_t *sd_table, SEXP_pstate_t *pstate, SEAP_scheme_t scheme, void *scheme_data);
+int          SEAP_desc_add (SEAP_desctable_t *sd_table, SEAP_scheme_t scheme, void *scheme_data);
```

Comment:
I would prefer to fix the indents to make the new code comply with our coding style guide, but that§s just a nitpick.

Reply:
I'm happy to fix this in a separate PR, but I'm not actually sure what the coding style guide is for OpenSCAP. IIRC `scap-workbench` uses QT style code, but I've seen so many different styles within OpenSCAP I'm not sure which is authoritative. 

Also, I'd prefer to fix the entire file at once rather than fixing just the line, especially since the next line has the same style and should be updated for consistency too. 

That's me though. If you want me to do it here, just tell me what the style guide is and I'll fix stuff :)

Reply:
We use Linux Kernel style https://www.kernel.org/doc/html/v4.10/process/coding-style.html
However some of the old code is not using it. We prefer to fix just the line that is being changed for another reason. We don't prefer big patches that change code style in whole file, because downstream Linux distributions and similar people have downstream patches which would become inapplicable. That's what we try to follow.

Reply:
Okay, thanks! I have updated this. :)